proto/p4/config/v1/p4info.proto

```diff
@@ -73,6 +86,10 @@ message PkgInfo {
   string url = 8;
   // Miscellaneous metadata, structured; a way to extend PkgInfo
   repeated StructuredAnnotation structured_annotations = 9;
+  // If set, specifies the properties that the underlying platform should have.
+  // If the platform does not conform to these properties, the server should
+  // reject the P4Info when used with a SetForwardingPipelineConfigRequest.
+  PlatformProperties platform_properties = 11;
```

Comment:
Are we missing `repeated SourceLocation annotation_locations = 10;` ?

Reply:
Looks like it's above, under annotations.

Reply:
Ahh, you're right, they're not in order, I forgot to expand the diff. Thanks!